### PR TITLE
Improvements/317 2

### PR DESCRIPTION
**Pull request by RomanGalochkin**
- Measurable Index bug has been fixed;
- FC lib has been synchronized;

### CODE REVIEW

File: packages/client/src/FC/Base/FC__Button.re [hunk 1]
```diff
@@ -77,39 +89,38 @@ let styles =
...
+      ~size=Large,
       ~isDisabled=false,
       ~fullWidth=false,
-      ~verticalPadding=`px(4),
       ~className="",
       children,
     ) => {
   ...component,
   render: _self =>
-    <FC__Link
```
Comment (OAGr) [inline packages/client/src/FC/Base/FC__Button.re]: I thought the FC__Link had some fancy-ish href properties. Are any of those needed?

File: packages/client/src/FC/Base/FC__Globals.re [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-        ),
+        lineHeight(`em(1.5)),
+      ],
+    ),
+    global(
+      "button",
+      [
+        fontFamily(FC__Settings.Text.standardFont),
+        fontSize(`px(16)),
+        lineHeight(`em(1.5)),
```
Comment (OAGr) [inline packages/client/src/FC/Base/FC__Globals.re]: Why are these being set here? Can we set them in in the button css instead? Having global properties is a bit of an anti-pattern, unless really needed.

File: packages/client/src/FC/Base/FC__Button.re (hunk 1 above)
Comment (RomanGalochkin) [reply packages/client/src/FC/Base/FC__Button.re]: Sorry, did not get properly.

File: packages/client/src/FC/Base/FC__Globals.re (hunk 2 above)
Comment (RomanGalochkin) [reply packages/client/src/FC/Base/FC__Globals.re]: The main client of the app has global settings which are imported with Antd lib, so this is why it had been placed here. Will change, thanks.

File: packages/client/src/pages/measurables/MeasurableIndex.re [hunk 3]
```diff
@@ -1,43 +1,36 @@
...
+              MeasurableIndex__Components.toLayoutInput(
+                reducerParams.send,
+                searchParams,
+                statsQuery,
+                MeasurableIndex__Logic.make({
+                  reducerParams,
+                  loggedInUser,
+                  channelQuery,
+                  seriesQuery,
+                }),
```
Comment (OAGr) [inline packages/client/src/pages/measurables/MeasurableIndex.re]: I was kind of proud of the ``load3Queries`` fn, but this is more compact. Maybe with hooks we should eventually be able to clean this up even more.

Comment (RomanGalochkin) [reply packages/client/src/pages/measurables/MeasurableIndex.re]: It was not readable, sorry, we should first of all write code for humans. Only after this refactoring I found a way to fix the issue.

File: packages/components/src/Charts/FC__CdfChart__Small.re [hunk 4]
```diff
@@ -31,8 +37,8 @@ let make =
       <FC__CdfChart__Base
         width=200
         height=40
-        minX
-        maxX
+        ?minX
+        ?maxX
```
Comment (OAGr) [inline packages/components/src/Charts/FC__CdfChart__Small.re]: +1 for making these optional.

File: packages/client/src/FC/Base/FC__Button.re (hunk 1 above)
Comment (OAGr) [reply packages/client/src/FC/Base/FC__Button.re]: Never mind; I thought it did some things it doesn't actually do.

File: packages/client/src/pages/measurables/MeasurableIndex.re (hunk 3 above)
Comment (OAGr) [reply packages/client/src/pages/measurables/MeasurableIndex.re]: sg

**Comment by OAGr** [comment]
I played with this locally, seems to work well!

**Comment by RomanGalochkin** [comment]
Updated, I have moved buttons styles in button component.

**Pull request merged**